data/110/873/259/5/1108732595.geojson

```diff
@@ -5,16 +5,16 @@
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
     "geom:area":0.012489,
-    "geom:area_square_m":93743898.142285,
+    "geom:area_square_m":93744087.925607,
     "geom:bbox":"90.033558,52.567262,90.213203,52.69528",
     "geom:latitude":52.625677,
     "geom:longitude":90.122365,
     "iso:country":"RU",
     "label:eng_x_preferred_longname":[
-        "Abaza District"
+        "Abaza Raion"
     ],
     "label:eng_x_preferred_placetype":[
-        "district"
+        "raion"
```

Comment:
Similar to some of the name properties.. worth moving the existing "district" value to a `label:eng_x_variant_placetype` property? Same goes for any label / name prop we're updating.

Reply:
Agree, I'll add a new commit...

Reply:
Reset back to `district` via https://github.com/whosonfirst-data/whosonfirst-data-admin-ru/pull/24/commits/66601ef140b14b95d1c32a7684b487fb2feee829.